packages/connector-discord/src/constant.ts

```diff
@@ -0,0 +1,46 @@
+import { ConnectorMetadata, ConnectorPlatform } from '@logto/connector-core';
+
+/**
+ * Base authorization URL.
+ * https://discord.com/developers/docs/topics/oauth2#shared-resources-oauth2-urls
+ */
+export const authorizationEndpoint = 'https://discord.com/oauth2/authorize';
+
+/**
+ * Discord exposes different versions of the API, You should specify which version to use by including it in your requests.
+ * https://discord.com/developers/docs/reference#api-reference
+ */
+export const accessTokenEndpoint = 'https://discord.com/api/v10/oauth2/token';
+export const userInfoEndpoint = 'https://discord.com/api/v10/users/@me';
+
+/**
+ * OAuth2 Scopes
+ * https://discord.com/developers/docs/topics/oauth2#shared-resources-oauth2-scopes
+ */
+export const scope = 'identify email';
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'discord-universal',
+  target: 'discord',
+  platform: ConnectorPlatform.Universal,
+  name: {
+    en: 'Discord',
+    'zh-CN': 'Discord',
+    'tr-TR': 'Discord',
+    'ko-KR': 'Discord',
```

Comment:
nit-picking: You can just define the name for `en` language only, since they are all the same and other locale will just fallback to English by default.